source/tools/testConsistency/dotnetWPF-renderer/WPFRenderer/WPFRenderer/Program.cs

```diff
@@ -0,0 +1,114 @@
+using System;
+using System.IO;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
I'm guessing that you probably don't use all of this `using` statements -- I think VS has a cleanup you can run to remove the ones you don't need :)